examples/unity-simple/Assets/Scripts/UdpEchoController.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright 2019 Google LLC
+// All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEngine;
+
+public class UdpEchoController : MonoBehaviour
+{
+    [SerializeField]
+    private GameObject server;
+    [SerializeField]
+    private GameObject client;
+
+    void Start()
+    {
+        server.SetActive(false);
+        client.SetActive(false);
+
+#if (UNITY_EDITOR || UNITY_SERVER)
+        server.SetActive(true);
+#endif
+#if !UNITY_SERVER

```

Comment:
Out of curiosity, where does this come from?

Reply:
`UNITY_SERVER` define is managed by Unity Build Settings.

* On GUI Build Settings.
  * check `Server Build`.
* Build by Script.
  * `BuildOptions.EnableHeadlessMode`.

Reply:
Very cool! I didn't know it was a thing!